tfhe/src/core_crypto/entities/lwe_ciphertext.rs

```diff
@@ -518,6 +518,18 @@ impl<T: UnsignedInteger, C: ContainerMut<Element = T>> AsMut<[T]> for LweCiphert
     }
 }
 
+// This accessor is used to create invalid objects and test the conformance functions
+// But these functions should not be used in other contexts, hence the `#[cfg(test)]`s
+#[cfg(test)]
+impl<C: Container> LweCiphertext<C>
+where
+    C::Element: UnsignedInteger,
+{
+    pub fn get_mut_ciphertext_modulus(&mut self) -> &mut CiphertextModulus<C::Element> {
```

Comment:
could these be made pub(crate) ? I think it's a pattern used for testing utilities (same for the other function in the other file) and you could update tfhe/src/core_crypto/entities/lwe_compact_ciphertext_list.rs as well as we forgot to make those function pub(crate) at the time, other than that I think the next iteration should be good to merge

Reply:
Done